core/templates/dev/head/editor/ExplorationEditor.js

```diff
@@ -172,6 +172,11 @@ oppia.controller('ExplorationEditor', [
     var _ID_TUTORIAL_PREVIEW_TAB = '#tutorialPreviewTab';
     var _ID_TUTORIAL_SAVE_BUTTON = '#tutorialSaveButton';
 
+    var screenWidth = windowDimensionsService.getWidth();
+    if (screenWidth <= 1280) {
+      _ID_TUTORIAL_PREVIEW_TAB = '#tutorialPreviewSmall';
```

Comment:
That sounds reasonable, too, though I just realized something: would this overwrite anything else bound to $window.onresize? If so, then the right design might be to register hooks with windowDimensionsService to be called one after the other on a resize event.

(/cc @giritheja for whom the same question applies)